src/p11_child/p11_child_openssl.c

```diff
@@ -44,8 +44,7 @@ struct p11_ctx {
     const char *ca_db;
     bool wait_for_card;
     struct cert_verify_opts *cert_verify_opts;
-    time_t start_timestamp;
-    time_t timeout;
+    int ocsp_deadline;
```

Comment:
Unlike `timeout`, this is a better fit for `time_t`, actually (see another comment).